traits/MLControl.php

```diff
@@ -156,7 +156,7 @@ public function getLocaleValue($locale)
             $value = $this->model->$mutateMethod($locale);
         }
         elseif ($this->objectMethodExists($this->model, 'getAttributeTranslated') && $this->defaultLocale->code != $locale) {
-            $value = $this->model->noFallbackLocale()->getAttributeTranslated($key, $locale);
```

Comment:
Please, keep the tests for the deprecated method (until it actually gets removed).